src/main/resources/schema/ans/0.10.0/story_elements/text.json

```diff
@@ -21,6 +21,9 @@
     "alignment": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_alignment.json"
     },
+    "format": {
```

Comment:
@gengel do you think it make sense to call it format? Maybe something like `block_properties`. Format kind of confusing with text format.

Reply:
Also, file name traits/trait_format.json sounds cryptic to me

Reply:
Agreed, just following everyone's file naming convention

Reply:
`block_properties` works...or `formatting`? I am open. This probably should've been more covered during proposal